dracut.sh

```diff
@@ -1668,10 +1668,12 @@
         mp=$(readlink -f "$dracutsysrootdir$mp")
         mountpoint "$mp" > /dev/null 2>&1 || continue
         _dev=$(find_block_device "$mp")
-        _bdev=$(readlink -f "/dev/block/$_dev")
-        [[ -b $_bdev ]] && _dev=$_bdev
-        [[ $mp == "/" ]] && root_devs+=("$_dev")
-        push_host_devs "$_dev"
+        if [[ $? -eq 0 ]]; then
```

Comment:
Not sure the best way to fix this since the command is also an assignment could do something like:
```
_dev=$(find_block_device "$mp") || is_fbd_error=1
if [[ $is_fbd_error -ne 1 ]]; then
```

I am not sure that would actually be better though

Reply:
For me the current way is ok, you just need to prepend `# shellcheck disable=SC2181` to avoid shellcheck complains.